burn-import/src/onnx/from_onnx.rs

```diff
@@ -173,6 +182,49 @@ fn move_initializer_data(initializer: &TensorProto, input: &mut Argument) {
     }
 }
 
+//this is an extremely hacky temporary solution while I figure out how to properly handle this
+//situation
```

Comment:
Will this be handled in this PR or later? We should replace with a TODO with instructions if not handled in this PR.

Also please explain the context why this is needed and if any potential issues it might cause down the road.